data/assets/scene/digitaluniverse/brown_dwarfs.asset

```diff
@@ -0,0 +1,69 @@
+local textures = asset.resource({
+  Name = "Brown Dwarf Textures",
+  Type = "HttpSynchronization",
+  Identifier = "digitaluniverse_brown_dwarfs_textures",
+  Version = 1
+})
+
+local speck = asset.resource({
+  Name = "Brown Dwarf Speck Files",
+  Type = "HttpSynchronization",
+  Identifier = "digitaluniverse_brown_dwarfs_speck",
+  Version = 4
+})
+
+
+local Object = {
+  Identifier = "BrownDwarfs",
+  Renderable = {
+    Type = "RenderablePointCloud",
+    Enabled = false,
+    File = speck .. "bd.speck",
+    Opacity = 1.0,
+    Texture = {
+      File = textures .. "point3.png",
+    },
+    Unit = "pc",
+    Coloring = {
+      FixedColor = { 0.4, 0.0, 0.1 },
+      ColorMapping = {
+        File = speck .. "bd.cmap",
+        ParameterOptions = {
+          { Key = "typeindex", Range = { 1.0, 4.0 } }
+        }
+      }
+    },
+    SizeSettings = {
+      ScaleExponent = 15.8,
+      MaxSize = 0.7,
+      EnableMaxSizeControl = true
+    }
+  },
+  GUI = {
+    Name = "Brown Dwarfs",
+    Path = "/Milky Way",
+    Description = [[A catalog of substellar objects, not large enough to be fully fledged stars, but larger than planets. Some are brown dwarfs, others are not massive enough for that moniker. Includes L, T, and Y dwarfs.]]
+  }
+}
+
+
+asset.onInitialize(function()
+  openspace.addSceneGraphNode(Object)
+end)
+
+asset.onDeinitialize(function()
+  openspace.removeSceneGraphNode(Object)
+end)
+
+asset.export(Object)
+
+
+
+asset.meta = {
+  Name = "Brown Dwarfs",
+  Version = "4.0",
```

Comment:
It is really a rename of an already existing file (was previously called just dwarfs.asset) and since the `asset.meta.name` hadn't changes, I ended up upping the version instead of setting it to 1.0

But I don't really know what's more/less confusing..

Reply:
Yes, it is the same data, albeit updated, but renamed. I'll defer to you all on how you want to handle that.